docs/knowledgebase/integrate/try-runtime.md

```diff
@@ -198,34 +201,37 @@ RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace \
 ## Next Steps
 
 ### Learn More
-- [Integrate `try-runtime`][integrate-tryruntime-htg] how-to guide  
```

Comment:
Are these links all broken, and actually meant to be used in the new docs site @sacha-l ? Can we address here to fix or open a new PR to resolve?